browser/src/components/Blob.jsx

```diff
@@ -33,10 +34,18 @@ export function BlobShaderMaterial({
       vertexShader,
     }),
     // eslint-disable-next-line react-hooks/exhaustive-deps
-    [],
+    []
   );
 
-  return <shaderMaterial attach="material" {...data} />;
+  useFrame((state) => {
```

Comment:
this will update the values every frame, why is it needed?

Reply:
uniform doesn't update properly without it 🤷 I'm not super familiar with `react-three-fiber` but this is the thread I was going off of https://spectrum.chat/react-three-fiber/general/update-uniforms-of-shadermaterial-when-props-change~8dddcbd1-59e3-4a5a-996b-7c460c8e57ff